src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java

```diff
@@ -254,6 +267,16 @@ public void serializeToView(@NotNull final ByteBuf buf)
         }
     }
 
+    public boolean shouldReplant(){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)